integrations/extensions/starter-kits/magnolia/README.md

```diff
@@ -0,0 +1,124 @@
+
+This starter kit exposes only the minimal functionality needed for simple use cases, and you will need to extend it if you want to cover more advanced ones.  For example, the API has parameters for restricting search based on the date or country that are not listed in the simple OpenAPI specification provided in the starter kit, so you would need to add these if you wanted to employ such filters.
+
+## Pre-Requisite Steps
+
+If you don't have a Delivery endpoint already setup to use with your Magnolia instance, make sure to follow the [Getting Started docs](https://docs.magnolia-cms.com/product-docs/6.2/Getting-started-with-Magnolia/Getting-started-with-REST.html) to setup one that allows you to get content from a certain workspace node.
+
+Since there is no security enabled by default on the REST endpoint, you should make sure to follow steps outlined in the [REST security](https://docs.magnolia-cms.com/product-docs/6.2/Modules/List-of-modules/REST-module/REST-security.html) section to provide restricted access to the endpoint only for an authorized user via their corresponding username/password.
+
+## Other Setup Info
+
+### Setup in a new Assistant
+
+If you want to make a _new_ Assistant using this starter kit, take the following steps:
+
+- Download the OpenAPI specification (`magnolia-search-openapi.json`) and Actions JSON file (`magnolia-search-actions.json`) in this starter kit.
+- Update the API path from `/delivery/tours/v1` to the endpoint setup by you for your Magnolia instance in the downloaded `magnolia-search-openapi.json`
+- Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+- [Add the extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension) using the username/password you setup for the API in the pre-requisites above.
+- [Upload the Actions JSON file](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import).
+- Use either method listed in [Configuring Your Actions Skill to use an Extension](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/README.md#configuring-your-actions-skill-to-use-an-extension) to configure the actions you uploaded to invoke the custom extension you built. Set the `q` parameter to the `query_text` session variable
+- Update the `workspace` variable to be the URL of the workspace that you have set up your REST endpoint for. Example: We use `http://34.71.156.142:8080/magnoliaPublic/tours` since the Delivery endpoint setup for this starter kit is querying the ***tours*** workspace
```

Comment:
I would replace "Update" with "In the Actions page go to `Variables > Created by you` and update" just to make it a little easier for users to find where to go.